doc/source/train/api.rst

```diff
@@ -117,10 +117,6 @@ Checkpointing
 
 .. _train-api-checkpoint-strategy:
 
-CheckpointStrategy
```

Comment:
Can we add `CheckpointConfig` to the API reference?

Reply:
it is in the api reference in air docs (all members of the `ray.air.config` package have their docs automatically generated)

Reply:
oh wait, it's not a member, duh. fixing